package.json

```diff
@@ -61,9 +64,11 @@
     "css-loader": "^5.2.7",
     "css-minimizer-webpack-plugin": "^3.4.1",
     "cssnano": "^5.0.17",
+    "cypress": "^7.4.0",
```

Comment:
on my machine installing cypress uses 7.7.0, I assume that is the latest, why do we use 7.4.0 please?

Reply:
@jomarko no reason, right now. I was not sure to put the latest version (9.6.0).